status-page.py

```diff
@@ -78,12 +84,15 @@ def update_incident_status(self):
                         'component_status': 2,
 
                     }
-                    print('Posting New Incident:')
-                    response = requests.post(f'{self.base_url}/incidents', headers=self.header, data=body)
-                    print(response)
+                    try:
+                        response = requests.post(f'{self.base_url}/incidents', headers=self.header, data=body)
+                    except requests.exceptions.RequestException as err:
```

Comment:
This exception seems used many times. I guess you can import it and directly use `RequestException`.